randovania/games/super_metroid/logic_database/Crateria.txt

```diff
@@ -895,6 +926,7 @@ Extra - asset_id: 27
           Any of the following:
               Grapple Beam or Space Jump
               Damage Boosting (Beginner) and Normal Damage ≥ 61
+              Horizontal Infinite Bomb Jump (Advanced) and Can Lay Bombs
```

Comment:
comment.
Also one is beginner, other advanced?
Same for DB

Reply:
Fixed